sympy/core/tests/test_equation.py

```diff
@@ -0,0 +1,72 @@
+from sympy import var, integrate, simplify, log, Integral
+from ..relational import Equality
+from ..function import *
+from ..equation import *
```

Comment:
In testing code I think it's better to have explicit absolute imports.

Reply:
I will work on this some more. Not all the explicit imports worked properly, when I tried them.

Reply:
Some imports may lead to circular import issues. In that case, you can generally fix it by moving the import inside of the function that uses it.

Reply:
done